docs/source/algorithms.md

```diff
@@ -4043,6 +4043,87 @@ these optimizers, you need to have
       initialization for speed. Default is False.
 ```
 
+```{eval-rst}
+.. dropdown:: nevergrad_oneplusone
+
+    .. code-block::
+
+        "nevergrad_oneplusone"
+
+    Minimize a scalar function using the One Plus One algorithm.
+
+    The Particle Swarm Optimization algorithm was originally proposed by
+    :cite:`Rechenberg1973`. The implementation in Nevergrad is based on
+    the one-fifth adaptation rule, going back to :cite:`Schumer1968`, discovered independently by :cite:`devroye1972` and :cite:`Rechenberg1973`.
+
+    An One Plus One evolutionary algorithm iterates to find a set of parameters that
+    produce the best possible registration result. It does this by 
+    perturbing, or mutating, the parameters from the last iteration (the
+    parent). If the new (child) parameters yield a better result, then 
+    the child becomes the new parent whose parameters are perturbed, 
+    perhaps more aggressively. If the parent yields a better result, it 
+    remains the parent and the next perturbation is less aggressive.
+
+    - **noise\_handling** (str or Tuple\[str, float] or None): Method for handling the noise. The name can be (Default: `None`).
+      - `"random"`: A random point is reevaluated regularly using the one-fifth adaptation rule.
+      - `"optimistic"`: The best optimistic point is reevaluated regularly, embracing optimism in the face of uncertainty.
+      - A float coefficient can be provided to tune the regularity of these reevaluations (default is 0.05).
```

Comment:
We need a bit more intuition. What is done with the result of the re-evaluation and how does it help in a noisy setting? What would a value of 0.05 mean? Is the re-evaluation done in 5 % of the iterations?